### PR TITLE
Dutch translation

### PR DESCRIPTION
**Pull request by m-burg**
I have translated everything except for the style guide.

When saving your preferences the confirmation still shows up as 'Saved changes', but I don not know enough web development to fix that.

Also some text boxes are a little too small for the words, this is noticeable for instance in the user settings.

### CODE REVIEW

File: shell/i18n/nl.i18n.json [hunk 1]
```diff
@@ -0,0 +1,1787 @@
...
+        "explanation": "Betaalde abonnees kunnen tot 30 GB gratis opslagruimte krijgen, 2 GB per persoon die ze verwijzen, naast de hoeveelheid opslag van hun account niveau. Eigenaren van gratis accounts kunnen een onbeperkte hoeveelheid korrels en 50MB opslagruimte per persoon die ze verwijzen verdienen, tot in totaal 2GB."
+      }
+    },
+    "about": {
+      "title": "Over Sandstorm",
+      "intro": {
+        "explanation": "<a href=\"https://sandstorm.io\">Sandstorm.io</a> is een open source hosting platform voor persoonlijke/privé instanties van web apps. Gebruikers kunnen software uploaden en installeren via een simpele app-store achtige interface. Naast betere privacy <a href=\"https://blog.sandstorm.io/news/2014-07-21-open-source-web-apps-require-federated-hosting.html\">is dit de enige manier om Open Source web apps haalbaar te maken.</a>",
+        "version": "Versie: %s",
+        "unofficialWarning": "<b>WAARSCHUWING:</b> deze server draait een onofficiële build van Sandstorm die aanpassingen kan bevatten die niet zijn gecontroleerd door de ontwikkelaars."
```
Comment (zenhack) [inline shell/i18n/nl.i18n.json]: It's in the existing localizations too, so orthoganal to this pr, but we should probably not be using the `<b>` tag. @kentonv, maybe we should change this to `<strong>`?

**Comment by kentonv** [comment]
Awesome, thanks!

File: shell/i18n/nl.i18n.json (hunk 1 above)
Comment (kentonv) [reply shell/i18n/nl.i18n.json]: I don't have a strong opinion (... so to speak), but would accept a pull request.

Comment (m-burg) [reply shell/i18n/nl.i18n.json]: I'll get on it

**Pull request merged**